### ISSUE TITLE
RC not listing on downloads page

### ISSUE THREAD

**Issue by tresf** [LMMS/lmms.io#265]
@lukas-w any idea?  Here's the report from Discord:

![screen shot 2018-07-30 at 12 38 31 am](https://user-images.githubusercontent.com/6345473/43377826-f1e675da-9390-11e8-86dd-0f1f3a05eecd.png)

**Comment by lukas-w** [comment]
No idea what's wrong. It works when I test it locally, so maybe deleting the caches will make it work again. I can't access the server through SSH any more though, maybe my key was removed?

**Comment by tresf** [comment]
> I can't access the server through SSH any more though, maybe my key was removed?

You're in there with a key... not sure if it's the right one.
```
ssh-rsa ... F7qtsv lukaswhl@gmail.com
ssh-rsa ... mOw== tresf
ssh-rsa ... 63DuF uros.maravic@gmail.com
ssh-ed25519 ... rql8c liushuyu@liushuyu
```

The session cleanup seems to be in bad shape again.

The source code that NetworkRedux wrote is:

Temp:
```bash
find /home/deploy/lmms.io/tmp -maxdepth 1 -type f -name 'sess*' -mtime +10 -exec rm -f {} \;
```

Wiki:
```bash
find /home/deploy/persistent/wiki/sessions -name sess* -mtime +10 -exec  rm -f {} \;
```

The wiki has a typo and hasn't been running, should be `-name 'sess*` -type f`.  That's fixed now but the downloads aren't.

I did a `sudo mv /tmp/github-api-cache/ /tmp/github-api-cache_bak` and it didn't fix it either.  Not sure what to try next.

**Comment by lukas-w** [comment]
> You're in there with a key

Thanks for checking. I restored that SSH key and can access the server again.

I think I know what's happening. @PhysSong started a draft of an RC7 release in the `lmms` repo three days ago. We're authenticated with the GitHub API so that rate limiting doesn't hit us, but apparently it makes draft releases show up in API responses as a side-effect. The website's code doesn't check for that and decides to display RC7 on the website – which it does, RC7 just doesn't have any downloads.

**Comment by lukas-w** [comment]
Fixed via 8258fbf2475f0aa2166a5c5e98952bcd6bbb3ed2.
